docker-compose.yaml

```diff
@@ -26,14 +26,14 @@ services:
   # nginx. The local project folder is mounted into the container as a volume to
   # allow development to be done on the host machine.
   app:
-    build: ./cbv3_django_prototype
+    build: ./project
     container_name: app
     restart: on-failure
     command: >
-      sh -c "python /opt/cbv3_django_prototype/manage.py migrate &&
-             uwsgi --ini /opt/cbv3_django_prototype/uwsgi.ini"
+      sh -c "python /opt/project/manage.py migrate &&
```

Comment:
This looks like it should reference `/opt/codebuddies/...`.

Reply:
Good catch, thanks!